components/connector/internal/gqlschema/schema.graphql

```diff
@@ -0,0 +1,59 @@
+# Token
+type Token {
+    url: String! # eg.: "https://connector-service.test.cluster.kyma.cx/v1/applications/info?token=1edfc34g"
+    token: String! # eg.: "1edfc34g"
+}
+
+# Certificate
+type Certificate {
+    crt: String!
+    caCrt: String!
+    clientCrt: String!
+}
+
+# MPInfo
+type MPInfo {
+    directorURL: String! # eg.: "https://director.cluster.kyma.cx/graphql"
+}
+
+# CSR
+type CSR {
+    subject: String! # eg.: "OU=Test,O=Test,L=Blacksburg,ST=Virginia,C=US,CN={ID}"
+    keyAlgorithm: String! # eg.: rsa2048
+}
+
+# CSRInfo
+type CSRInfo {
+    csrURL: String! # eg.: "https://connector-service.test.cluster.kyma.cx/v1/runtimes/certificates?token=1edfc34g"
```

Comment:
hmm, maybe we should put this URL into `CSR` type.

Reply:
Also I would rather not use abbreviations for type names. I would go with the `CertificateSigningRequest` name

Reply:
After further considerations, this URL is not relevant here, so I remove it.